pkg/filesystem/fs/fs.go

```diff
@@ -143,11 +143,11 @@ func NewFileSystem(ctx context.Context, opt ...NewFSOpt) (*Filesystem, error) {
 	return &fs, nil
 }
 
-func (fs *Filesystem) CleanupBlobLayer(ctx context.Context, key string, async bool) error {
+func (fs *Filesystem) CleanupBlobLayer(ctx context.Context, id string, async bool) error {
```

Comment:
`id` -> `blobDigest` is better.

Reply:
Fixed.
The reason I didn't use blobDigest is that i thought, the type of blobDigest is preferably digest.Digest in general.